api/envoy/service/status/v3/csds.proto

```diff
@@ -79,6 +79,8 @@ message PerXdsConfig {
     admin.v3.RoutesConfigDump route_config = 4;
 
     admin.v3.ScopedRoutesConfigDump scoped_route_config = 5;
+
+    admin.v3.EndpointsConfigDump endpoint_config = 6;
```

Comment:
Should this be `[#not-implemented-yet:]`?

Reply:
Sorry about that, solved